xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-api/src/main/java/org/xwiki/search/solr/internal/metadata/DocumentSolrMetadataExtractor.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.solr.common.SolrInputDocument;
 import org.xwiki.component.annotation.Component;
 import org.xwiki.model.EntityType;
+import org.xwiki.model.document.DocumentAuthors;
```

Comment:
Note that I can remove this import for reducing the fan-out complexity, but I personally think it makes the code a bit less readable. And it's not enough to get rid of the checkstyle problem.